apps/judicial-system/web/src/routes/CourtOfAppeal/AppealCase/appealCaseUsers.graphql

```diff
@@ -18,6 +18,7 @@ query AppealCaseUsers {
       active
     }
     active
+    canConfirmAppeal
```

Comment:
Do we use this information here? Consider only adding it to the graphql queries where the field is used